src/lib.rs

```diff
@@ -1,3 +1,91 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use serde_derive::{Deserialize, Serialize};
+use std::sync::Arc;
+use tokio::sync::Mutex;
+use uuid::Uuid;
+
+pub const LOCAL_LISTEN_ADDRESS: &str = "192.168.1.202";
+
+pub const PROTO_VERSION: f32 = 0.1;
+pub const PROTO_NAME: &str = "Enarx-Keep-Manager";
+pub const BIND_PORT: u16 = 3030;
+
+pub const KEEP_LOADER_STATE_UNDEF: u8 = 0;
+pub const KEEP_LOADER_STATE_LISTENING: u8 = 1;
+pub const KEEP_LOADER_STATE_STARTED: u8 = 2;
+pub const KEEP_LOADER_STATE_COMPLETE: u8 = 3;
+pub const KEEP_LOADER_STATE_ERROR: u8 = 15;
+
+pub const KEEP_INFO_COMMAND: &str = "keep-info";
+pub const CONTRACT_COMMAND: &str = "command";
+pub const KEEP_COMMAND: &str = "command";
+pub const KEEP_AUTH: &str = "auth-token";
+pub const KEEP_PORT: &str = "keep-port";
+pub const KEEP_ADDR: &str = "keep-addr";
+pub const KEEP_KUUID: &str = "kuuid";
+pub const KEEP_ARCH: &str = "keep-arch";
+pub const WASMLDR_BIND_PORT_CMD: &str = "wasmldr-bind-port";
+pub const WASMLDR_ADDR_CMD: &str = "wasmldr-addr";
```

Comment:
Are these meant to be MIME-type packet IDs?

Reply:
No - they're what I've been using till now.  I'm going to keep them as is for now until we start transitioning.